src/amcl/.vscode/settings.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Oops. This file probably shouldn't be checked in.

Reply:
Fixed.